taxonomies/additives.txt

```diff
@@ -6924,7 +6924,7 @@ es:E316, eritorbato sódico
 et:E316, Naatriumerütorbaat
 fa:E316, سدیم اریتوربات
 fi:E316, Natriumerytorbaatti, Natriumerytorbaattia
-fr:E316, érythorbate de sodium, isoascorbate de sodium
+fr:E316, érythorbate de sodium, isoascorbate de sodio
```

Comment:
That's the entry for French, not Spanish

Reply:
Fixed